.github/workflows/yocto-build.yml

```diff
@@ -3,11 +3,19 @@ name: "Yocto"
 on:
   workflow_call:
     secrets:
+      # API key names should be descriptive telling us what company, what for, where from, what's the key.
+      # Let's say, XCompany wants to have test, then keys could be: XCOMPANY_AUTOKIT_BM_ACCESS_TOKEN
+      AUTOKIT_BALENACLOUD_API_KEY:
+        description: balena API key for Leviathan testing from BalenaCloud
+        required: false
+      AUTOKIT_BALENAMACHINE_API_KEY:
+        description: balena API key for Leviathan testing from balena-os BM instance
+        required: false
```

Comment:
Do we need these to be separate? Why not use a single secret and have ops change it if/when we move autokit back to balenaCloud.

What about `AUTOKIT_FLEET_API_KEY` so we know what it's used for but is environment agnostic?

Reply:
The issue at the moment @klutchell is that we have a split fleet, some autokits are balena cloud, some balena machine. I agree these shouldn't be separate though

I think what we actually want here is a step further down, that selects the correct API key for the environment

Reply:
That's unfortunate. What we should do then, is add `environment` to the Leviathan test matrix so when we build the matrix we will populate a value for environment then the appropriate vars can be read in the test run.

This doesn't solve secrets unfortunately unless we add them manually to all the environments, or GitHub starts supporting org-level environments.